CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEfficiencyHarvester.cc

```diff
@@ -616,39 +621,39 @@ void SiStripHitEfficiencyHarvester::makeSummary(DQMStore::IGetter& getter,
       LOGPRINT << "Fill only good modules layer " << i << ":  S = " << goodlayerfound[i] + goodlayerfound[i + 3]
                << "    B = " << goodlayertotal[i] + goodlayertotal[i + 3];
       if (goodlayertotal[i] + goodlayertotal[i + 3] > 5) {
-        found->setBinContent(i, goodlayerfound[i] + goodlayerfound[i + 3]);
-        all->setBinContent(i, goodlayertotal[i] + goodlayertotal[i + 3]);
+        found_good->setBinContent(i, goodlayerfound[i] + goodlayerfound[i + 3]);
+        all_good->setBinContent(i, goodlayertotal[i] + goodlayertotal[i + 3]);
       }
       LOGPRINT << "Filling all modules layer " << i << ":  S = " << alllayerfound[i] + alllayerfound[i + 3]
                << "    B = " << alllayertotal[i] + alllayertotal[i + 3];
       if (alllayertotal[i] + alllayertotal[i + 3] > 5) {
-        found2->setBinContent(i, alllayerfound[i] + alllayerfound[i + 3]);
-        all2->setBinContent(i, alllayertotal[i] + alllayertotal[i + 3]);
+        found_all->setBinContent(i, alllayerfound[i] + alllayerfound[i + 3]);
+        all_all->setBinContent(i, alllayertotal[i] + alllayertotal[i + 3]);
       }
     }
     for (unsigned int i = 17; i < 17 + nTEClayers_; ++i) {  // TEC disks
```

Comment:
I'm curious, even though I have no reason to believe that this is uncorrect: TEC layers end at 22 (see `bounds::k_LayersAtTECEnd`): why is it "<17 + nTEClayers_", where  `nTEClayers_` can be either 7 or 9, here?
Shouldn't those constanst be preferably used here, instead of these "magic numbers"?

Reply:
because the amount of rings / disks to be counted is halved in case the two sides are a folded in the same counter.
I agree this code is full of hardcoded geomety constants, but I'd rather not touch them now at the risk of introducing errors (in case this is clearly beyond the scope of the PR).